docs/IModelsClientAuthoring.md

```diff
@@ -0,0 +1,78 @@
+# `@itwin/imodels-client-authoring`
+
+`IModelsClient` exposed by `@itwin/imodels-client-authoring` package extends `IModelsClient` exposed by `@itwin/imodels-client-management` package so this documentation references sections from [`@itwin/imodels-client-management` documentation](IModelsClientManagement.md). All types mentioned there are also exported from `@itwin/imodels-client-authoring` package.
+
+## Key types
+- [`IModelsClient`](../clients/imodels-client-authoring/src/IModelsClient.ts#L31)
+- [`IModelsClientOptions`](../clients/imodels-client-authoring/src/IModelsClient.ts#L18)
+
+### Parameter and response types
+Please see [parameter and response types for `@itwin/imodels-client-management`](./IModelsClientManagement.md#parameter-and-response-types).
```

Comment:
Please see [parameter and response types inherited from `@itwin/imodels-client-management` package](./IModelsClientManagement.md#parameter-and-response-types).